apiserver/plane/api/views/issue.py

```
@@ -314,6 +315,33 @@
+                    item["bool_value"] = bool_value
+                elif item.get("data_type") == "date":
+                    raw_date = item.get("value")
+                    try:
+                        # Try to parse date in standard ISO format
+                        date_value = datetime.strptime(raw_date, "%Y-%m-%d").date()
+                    except (ValueError, TypeError):
+                        date_value = None  # Invalid or missing date
+                    item["date_value"] = date_value
+
+        request.data["custom_properties"]= data
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)